Sources/OpenAI/Public/Service/DefaultOpenAIService.swift

```diff
@@ -85,7 +87,6 @@ struct DefaultOpenAIService: OpenAIService {
    {
       var chatParameters = parameters
       chatParameters.stream = true
-      chatParameters.streamOptions = .init(includeUsage: true)
```

Comment:
@lzell 👀
I moved this to the parameters initializer, Some OpenAI compatible API's like Gemini will fail if we pass this.

Reply:
Thank you @jamesrochabrun !